badges/src/main/java/com/decathlon/vitamin/compose/badges/VitaminBadgeSizes.kt

```diff
@@ -47,10 +48,10 @@ object VitaminBadgeSizes {
     @Composable
     fun large(
         horizontalPadding: Dp = 6.dp,
-        verticalPadding: Dp = 5.dp,
+        verticalPadding: Dp = 0.dp,
```

Comment:
It is weird no? The previous implementation was wrong?

Reply:
Same comment for other variants

Reply:
According to the figma specifications :
<img width="196" alt="image" src="https://user-images.githubusercontent.com/46709142/202714936-507c1b43-bebb-45fd-94ae-2a49337d2c29.png">
This component has no vertical padding.